machine/accton/accton_as7312_54x/busybox/conf/config

```diff
@@ -6,3 +6,7 @@ CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=8
 CONFIG_SYS_EEPROM_OFFSET=0
 CONFIG_SYS_EEPROM_SIZE=256
 CONFIG_SYS_EEPROM_MAX_SIZE=256
+CONFIG_MKSWAP=y
+CONFIG_FEATURE_MKSWAP_UUID=y
+CONFIG_SWAPONOFF=y
+CONFIG_FEATURE_SWAPON_PRI=y
```

Comment:
@cbrune, Happy Lunar New Year!
I'm not sure whether most NOS installers need these commands or not.  Currently, we have only two machines for the same customer that needs the commands to create swap partition.  I think it can to be available to every platform, too.